New abilities first commit (#36)
GameBrains/er-core: modules/experimental/pages/ff-blast.adoc

= FF Blast

`Detachment ability`

Some forces are able to disrupt their firefight opponents, for example by using the sonic weapons of Slaaneshi Noise Marines.

> 

Reviewer: ```suggestion

[NOTE]
====
'CC Blast' and 'FF Blast' are temporary names -- we hope to settle on better names for these abilities in future.
In a perfect world, these abilities will have names that are evocative and yet which also neatly hint at the relatedness of their distinct yet similar gameplay effects. Easier said than done!
====

```

Author: OK.

---

Where your detachment has the {ff-blast} ability and takes part in a firefight:

> * After rolling to resolve the firefight, every enemy detachment that was within 15 cm of your detachment gains one Blast marker. This effect does not stack with other detachments with the {ff-blast} ability.

Reviewer: I thought I had an edge case to consider here, but it looks like line of fire doesn't come into the definition of 'engaged', so that keeps things simpler, phew! :-D

So, just a little neatening:

```suggestion
* After you roll for assault resolution in this firefight, place one additional Blast marker on each enemy detachment that was within 15 cm of this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```

Author: As with CC Blast, 'additional' isn't appropriate.

If we refer directly to the step again:

```suggestion
* At the end of xref:roll-to-resolve-the-firefight.adoc[], place one Blast marker on each enemy detachment that was within 15 cm of this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```

Reviewer: Yup, I take your point again. Similar to our discussion on CC Blast:

```suggestion
* At the end of the xref:main-rules:roll-to-resolve-the-firefight.adoc[] step, place one Blast marker on each enemy detachment that had any units within 15 cm of units in this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```

= FF Blast

`Detachment ability`

> Some forces are able to disrupt their firefight opponents, for example by using the sonic weapons of Slaaneshi Noise Marines.

Reviewer: ```suggestion
These units have some special means to disrupt their opponents in firefights.

For example, Noise Marines of Slaanesh use their psycho-sonic weapons to produce 'Kakophoni' effects that are both murderous and disorienting.
```
